pkg/library/libraryapplyconfiguration/client_mutations.go

```diff
@@ -22,7 +24,9 @@ func (s *clientBasedClusterApplyResult) Requests() MutationActionReader {
 	return s.mutationTracker
 }
 
-func NewApplyConfigurationFromClient(mutationTracker *manifestclient.AllActionsTracker[manifestclient.TrackedSerializedRequest]) *applyConfiguration {
+func NewApplyConfigurationFromClient(
```

Comment:
This function appears to be assigning the mutationTracker, which holds all serialized requests, to all types of clusters. 

However, I don't see where the mapping of serialized requests to specific cluster types is being done.

Reply:
> However, I don't see where the mapping of serialized requests to specific cluster types is being done.

15 lines, down in `FilterAllDesiredMutationsGetter` https://github.com/openshift/multi-operator-manager/pull/16/files#diff-7c6bf328fe10881be715bb2460cc7d1628316f4aa43b99438274b53d270768e0R43 called from the command: https://github.com/openshift/multi-operator-manager/pull/16/files#diff-817f4737e0c52096f9428fd232ea539f40b2885a60d93b387bfd86bc1c654a66R56

This way each operator doesn't have to get it just right.

Reply:
and the outputs themselves are here: https://github.com/openshift/cluster-authentication-operator/blob/master/pkg/cmd/mom/output_resources_command.go#L16-L43

Reply:
OK, thanks for the pointers.